automation/tdx_canonical_verifier.sh

```diff
@@ -19,6 +18,7 @@ if [[ -z "$SUDO_USER" ]]; then
 else
 	LOGIN_USER=$SUDO_USER
 fi
+TD_IMAGE_DIR=/home/$LOGIN_USER/td_image
```

Comment:
Since you are always just accessing the full image path, do you want to define this variable as `TD_IMAGE_PATH` and set it to `/home/$LOGIN_USER/td_image/tdx-guest-ubuntu-24.04-generic.qcow2`?